WordPress/Classes/Utility/WPWebViewController.m

```diff
@@ -318,6 +318,7 @@ - (void)setUrl:(NSURL *)theURL
 
 - (IBAction)dismiss
 {
+    [ReachabilityUtils dismissNoInternetConnectionNotice];
```

Comment:
ℹ️ What do you think about moving this in `viewWillDisappear` so we're consistent with the other places where we dismiss Notices? For example, in `ReaderStreamViewController`, `AbstractPostListViewController`, and in `WebKitViewController`, we call this in `viewWillDisappear`. Also, `viewWillDisappear` is safer in the unlikely case that there's another way to dismiss `WPWebViewController`.

Reply:
Sure thing